exporter/awsxrayexporter/awsxray_test.go

```diff
@@ -82,7 +81,8 @@ func BenchmarkForTracesExporter(b *testing.B) {
 		ctx := context.Background()
 		td := constructSpanData()
 		b.StartTimer()
-		traceExporter.ConsumeTraces(ctx, td)
+		err := traceExporter.ConsumeTraces(ctx, td)
+		assert.NotNil(b, err)
```

Comment:
```suggestion
		assert.Error(b, err)
```

Carry this and `assert.NoError()` changes througout.